src/components/swap/SwapBox.vue

```diff
@@ -43,7 +44,7 @@
           <span class="text-left ml-4">receive</span>
           <div class="currency-input">
             <div class="input-holder">
-              <input placeholder="0.0" v-model="swapTo.value" />
+              <input placeholder="amount to receive" v-model="swapTo.value" />
```

Comment:
At a first sight looks good, but please rebase from main so I can test these changes integrated to the latest ones being merged into main.